src/interventions/Interventions.jl

```diff
@@ -117,6 +129,7 @@ Base.@kwdef struct Intervention{N,P,P2} <: EcoModel
         bounds=(2.0, 25.0 + 1.0),
         default_bounds=(2.0, 25.0 + 1.0),
         dists="unif",
+        criteria_keywords=(""),
```

Comment:
You're aware that this creates an empty string?

If you want a tuple with a single element, it is `("", )`